### PR TITLE
camerarecord: add possibility to select slow_motion recording mode

### PR DESCRIPTION
**Pull request by mdigiorgio**
Signed-off-by: Michele Di Giorgio <michele.digiorgio@arm.com>

### CODE REVIEW

**Comment by setrofim** [comment]
The code looks good. Are you reasonably sure this will work across different android versions (at least KitKat onwards), or is a version check needed?

**Comment by mdigiorgio** [comment]
I've implemented this feature only inside the `recordVideoGoogleV3_2()` and tested it on the N5X (so Android M). I'm not hundred percent sure about previous versions, are you able to give it a try? I don't have other devices on which to test this.

**Comment by setrofim** [comment]
Yes, you're right, that will only be invoked for M onwards, so it should be fine. However you should add a check in `setup` of the workload to raise a `WorkloadError` if `self.device.get_android_sdk_version()` is less than 23.

**Comment by mdigiorgio** [comment]
OK, I provided a check for that in `camerarecord/__init__.py:setup()`.

File: wlauto/workloads/camerarecord/__init__.py [hunk 1]
```diff
@@ -48,8 +51,14 @@ def initialize(self, context):
         version = self.device.get_installed_package_version(self.package)
         version = version.replace(' ', '_')
         self.uiauto_params['version'] = version
+        self.uiauto_params['recording_mode'] = self.recording_mode
 
     def setup(self, context):
+        if self.recording_mode != 'normal':
+            if self.device.get_sdk_version() < 23:
+                raise WorkloadError('{} recording mode only supported by '\
```
Comment (setrofim) [inline wlauto/workloads/camerarecord/__init__.py]: Format specifier never used.

Have you run pylint/pep8 checkers for this?

Comment (mdigiorgio) [inline wlauto/workloads/camerarecord/__init__.py]: Oops, forgot it sorry. Going to change it and push again.

**Pull request merged**